prime-router/metadata/schemas/covid-19.schema

```diff
@@ -683,6 +699,22 @@ elements:
     documentation: |
       The patient's race. There is a common valueset defined for race values, but some states may choose to define different code/value combinations.
 
+  - name: patient_race_id
```

Comment:
The other way to do this would be for hl7-ingest-covid-19 schema to override the `valueSet: hl7005` in `patient_race` with `valueSet: whatever`, then define a `whatever` Valueset.    Why not do it that way, instead of adding a new mapper( ...)?

This implementation trades off flexibility with readability.   Each row of data can potentially come from a different type system.  However, we'll no longer have a way to go into XYZ's schema and know what valueset XYZ is using for patient_race.

The design around this way of managing multiple valuesets was meant to be one of the major topics in #1468:   Let's say there are 50 different ways senders/receivers have to define the possible values for patient_race -- how do we manage that?  What tools do we have for translating between different value sets?

Reply:
Removed these changes as discussed since it needs a lot more thought behind it.  Will leave for #1468